build.yaml

```diff
@@ -11,23 +11,49 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 steps:
+
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'build-frontend-buildx'
+  args: ['buildx', 'create', '--name=buildx-frontend-builder', '--driver=docker-container', '--driver-opt', 'network=cloudbuild', '--use']
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'build-frontend'
+  waitFor: ['build-frontend-buildx']
+  args: ['buildx', 'build', '-f', 'cmd/frontend/Dockerfile', '.', '--platform', 'linux/arm64,linux/amd64', '-t', 'gcr.io/$PROJECT_ID/prometheus-engine/frontend:$TAG_NAME', '--push']
+  timeout: 7200s
+
+
+
+- name: 'gcr.io/cloud-builders/docker'
+  id: 'build-config-reloader-buildx'
+  args: ['buildx', 'create', '--name=buildx-config-reloader-builder', '--driver=docker-container', '--driver-opt', 'network=cloudbuild', '--use']
```

Comment:
Do we need to create a different bulidx builder for each image or can we reuse the one already created?

Reply:
if we want to parallel the build, each build need to have a separate buildx builder.